lesson_25/jimoye244a/src/pages/Home/components/program-list/ProgramList.scss

```diff
@@ -0,0 +1,11 @@
+.programs-section {
```

Comment:
Seems like some styles are missing in this or the `Program` component. However, you've got the gist of how this is supposed to work.